lib/thumbs/pull_request_worker.rb

```diff
@@ -702,7 +702,10 @@ def create_build_status_comment
 
 <%= status[:command] %>
 
-<%= status[:output] %>
+<% output=status[:output] %>
+<% allowed_length=10000 %>
+<%= output.slice!(output.length-allowed_length, output.length) %>
```

Comment:
`ellipses = status[:output].length > allowed_length ? ' ...' : ''` or something of the like should be computed then appended to the output so it is clear that the content was clipped. NOTE: the slice would also need to subtract out the ellipses length, so `output.slice!(output.length - ellipses.length - allowed_length, output.length)` .